hs_tools_resource/models.py

```diff
@@ -85,16 +85,28 @@ def get_supported_res_types_str(self):
         return ','.join([parameter.description for parameter in self.supported_res_types.all()])
 
     @classmethod
+    def _add_supported_res_type(cls, meta_instance, res_type):
+        if isinstance(res_type, int):
+            # "copy res" or "create a new version"
+            qs = SupportedResTypeChoices.objects.filter(id=res_type)
+            if not qs.exists():
+                raise
+            meta_instance.supported_res_types.add(qs[0])
```

Comment:
Do we need to do this add here? Also why you do this if statement here? I think if qs is not exist it will be created at line 101.